services/QuillLMS/.rubocop_todo.yml

```diff
@@ -67,65 +67,6 @@ Layout/CommentIndentation:
   Exclude:
     - 'app/controllers/schools_controller.rb'
 
-# Offense count: 2
-# This cop supports safe autocorrection (--autocorrect).
-Layout/EmptyLineAfterGuardClause:
-  Exclude:
-    - 'engines/evidence/lib/generators/quill_model/quill_model_generator.rb'
-    - 'engines/evidence/spec/dummy/app/models/activity.rb'
-
-# Offense count: 34
-# This cop supports safe autocorrection (--autocorrect).
-# Configuration parameters: EmptyLineBetweenMethodDefs, EmptyLineBetweenClassDefs, EmptyLineBetweenModuleDefs, DefLikeMacros, AllowAdjacentOneLineDefs, NumberOfEmptyLines.
-Layout/EmptyLineBetweenDefs:
```

Comment:
What is the behavior of this rule in the context of endless methods? I’d like to retain the ability to have many endless methods without whitespace between them

Reply:
It does not require spaces between endless methods.